calico_node/filesystem/etc/calico/confd/templates/bird_ipam.cfg.template

```diff
@@ -2,20 +2,29 @@
 filter calico_pools {
   calico_aggr();
   custom_filters();
-{{range ls "/pool"}}{{$data := json (getv (printf "/pool/%s" .))}}
+{{range ls "/v1/ipam/v4/pool"}}{{$data := json (getv (printf "/v1/ipam/v4/pool/%s" .))}}
   if ( net ~ {{$data.cidr}} ) then {
     accept;
   }
 {{end}}
   reject;
 }
 
+{{$network_key := printf "/bgp/v1/host/%s/network_v4" (getenv "NODENAME")}}{{$network := getv $network_key}}
 filter calico_ipip {
```

Comment:
I'd like to see some comments in this template to explain a bit about what each branch is meant to do.

I think I understand the flow, but it takes a bit of reading to figure it out without any comments.